tests/python/unittest/test_loss.py

```diff
@@ -180,8 +180,7 @@ def test_l1_loss():
     assert mod.score(data_iter, eval_metric=mx.metric.Loss())[0][1] < 0.1
 
 
-@unittest.skip("flaky test. https://github.com/apache/incubator-mxnet/issues/8892")
-@with_seed()
+@with_seed(1234)
```

Comment:
Please don't fix the seed if possible

Reply:
Wait. This code does not make use of random data at all, does it? So why do we have to set a seed?